woodpecker-go/woodpecker/repo.go

```diff
@@ -361,13 +368,6 @@ func (c *client) PipelineDecline(repoID, pipeline int64) (*Pipeline, error) {
 	return out, err
 }
 
-// PipelineKill force kills the running pipeline.
```

Comment:
This one has confused me. IMO naming it `PipelineKill` doesn't make sense and deleting the pipeline as "force kill" action is strange somehow. Any idea why it was implemented that way?

Reply:
Looks like nobody know. However, I've renamed this method now.